packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +136,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? const ClampingScrollPhysics(),
```

Comment:
er, other way around I guess. `physics.applyTo(const ClampingScrollPhysics())`. Whichever way makes the physics that was passed in take priority. You'd still need to explicitly call out that you shouldn't override the clamping method, though.